ruby/lang/reflection_using_user_input.yml

```diff
@@ -82,6 +82,8 @@ patterns:
         values:
           - constantize
           - safe_constantize
+          - classify.constantize
```

Comment:
The `USER_INPUT` filter should be considering child nodes, so I'd expect it to match without needing these additions.

Reply:
Ah, so it is 🙈 I'll close this one. Sorry for the noise!